.gitignore

```diff
@@ -1,14 +1,15 @@
-## Ignore build results and temporary files.
```

Comment:
Weird, I can't tell what the whitespace change is. Since the new file is bigger I'm going to guess the new one has windows line endings.

I did the https://github.com/TabbedOut/websocket-sharp/pull/3/files?w=1 get parameter trick to hide whitespace changes